db/R/get.trait.data.R

```diff
@@ -272,15 +272,16 @@ runModule.get.trait.data <- function(settings) {
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) & settings$meta.analysis$update
+    if ((forceupdate == 'AUTO') || !as.logical(forceupdate))
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
     settings$pfts <- get.trait.data(pfts, modeltype, dbfiles, database, forceupdate)
     return(settings)
   } else if(is.Settings(settings)) {
     pfts <- settings$pfts
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) &settings$meta.analysis$update
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
```

Comment:
oh I see, good to know.

Then we won't meet [this condition](https://github.com/PecanProject/pecan/blob/master/db/R/get.trait.data.R#L86) because `forceupdate` is set to `TRUE` [here](https://github.com/PecanProject/pecan/blob/master/db/R/get.trait.data.R#L283) and it cannot be `AUTO`. This way it doesn't throw an error in TRAIT step but also it's not running through what it is supposed to be. 

I believe previously we were setting it to "AUTO", not sure at which point conditional checks were put there
```
forceupdate <- settings$meta.analysis$update 
```